private.tf

```diff
@@ -12,14 +12,18 @@ module "private_subnet_label" {
   source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.3"
   namespace = "${var.namespace}"
   stage     = "${var.stage}"
-  name      = "private"
+  name      = "${var.name}-private"
```

Comment:
I'm torn here.
On the one hand, we need to follow the pattern and add `name` to every resource in a project.
In this case, it's better to use `attributes` and not hardcode name prefixes:

```
  name       = "${var.name}"
  attributes = ["private"]
```

but on the other hand, the subnets are for the entire infrastructure and many different applications could be placed into the subnets (e.g. app servers, Postgres, Redis, etc.), all with their own different names.
So here we are talking about just the names of the subnets itself.
If we follow the same pattern, then we could do this
```
  name       = "subnet"
  attributes = ["private"]
```
But we do not use the resource types in resource names.
That's why we just named the subnets `private` and `public`.